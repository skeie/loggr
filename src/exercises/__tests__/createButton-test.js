import React from 'react';
import renderer from 'react-test-renderer';
import CreateButton from '../CreateButton';
test('Should render CreateButton correctly', () => {
    const tree = renderer.create(
        <CreateButton /> 
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
    