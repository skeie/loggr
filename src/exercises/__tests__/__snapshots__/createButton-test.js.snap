exports[`test Should render CreateButton correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "bottom": 0,
      "justifyContent": "center",
      "position": "absolute",
      "width": 750,
    }
  }>
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    hitSlop={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}>
    <Image
      source={1}
      style={
        Object {
          "alignSelf": "center",
          "marginBottom": 20,
        }
      } />
  </View>
</View>
`;
